contracts/cw-pre-propose-base-proposal-single/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "cw-pre-propose-base-proposal-single"
```

Comment:
Fine for a name, but when we reorg the folder structure it may be good to put these all in a `pre-propose` folder.

Reply:
yep! planning on doing that. need to split do renaming fist because currently if we have a folder `contracts/pre-propose` cargo will read that as a rust project (not a folder of rust projects) as we specify `contracts/*` as our workspace. the renaming addresses this by moving every module into a subfolder with that module's implementations so that we can specify `contracts/proposal-modules/*, contracts/pre-propose/*` in our `Cargo.toml`.